Traceback (most recent call last):
  File "C:\Users\nares\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\nares\anaconda3\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\nares\anaconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\nares\anaconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\nares\anaconda3\lib\asyncio\base_events.py", line 642, in run_until_complete
    return future.result()
  File "C:\Users\nares\anaconda3\lib\site-packages\nbclient\client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "C:\Users\nares\anaconda3\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\nares\anaconda3\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
from typing import List, Tuple
import cv2
from google.colab.patches import cv2_imshow

from cv2 import cvtColor, COLOR_BGR2GRAY, TERM_CRITERIA_EPS, TERM_CRITERIA_MAX_ITER, \
    findChessboardCorners, cornerSubPix, drawChessboardCorners

'''
Please do Not change or add any imports. 
'''

img_given = cv2.imread('checkboard.png')
cv2_imshow(img_given)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_16616/3544235127.py[0m in [0;36m<module>[1;34m[0m
[0;32m      2[0m [1;32mfrom[0m [0mtyping[0m [1;32mimport[0m [0mList[0m[1;33m,[0m [0mTuple[0m[1;33m[0m[1;33m[0m[0m
[0;32m      3[0m [1;32mimport[0m [0mcv2[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 4[1;33m [1;32mfrom[0m [0mgoogle[0m[1;33m.[0m[0mcolab[0m[1;33m.[0m[0mpatches[0m [1;32mimport[0m [0mcv2_imshow[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      5[0m [1;33m[0m[0m
[0;32m      6[0m [1;32mfrom[0m [0mcv2[0m [1;32mimport[0m [0mcvtColor[0m[1;33m,[0m [0mCOLOR_BGR2GRAY[0m[1;33m,[0m [0mTERM_CRITERIA_EPS[0m[1;33m,[0m [0mTERM_CRITERIA_MAX_ITER[0m[1;33m,[0m[0;31m [0m[0;31m\[0m[1;33m[0m[1;33m[0m[0m

[1;31mModuleNotFoundError[0m: No module named 'google.colab'
ModuleNotFoundError: No module named 'google.colab'

